### PR TITLE
[MM-59346] Add labels for channel header and purpose in RHS channel info view

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-59346

#### Release Note

```release-note
Added labels for channel header and purpose in RHS channel info view
```

### CODE REVIEW

File: webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx [hunk 1]
```diff
@@ -27,6 +27,18 @@ const ChannelPurpose = styled.div`
     }
 `;
 
+const ChannelDescriptionHeading = styled.div`
+    color: var(--center-channel-color-75, rgba(63, 67, 80, 0.75));
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx:31]: we dont need this fallback
```
color: rgba(var(xxx...
```

File: webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx [hunk 2]
```diff
@@ -27,6 +27,18 @@ const ChannelPurpose = styled.div`
...
+    font-family: "Open Sans";
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx:32]: do we need to specifically add this style, arent font inherited?

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
i18n check failed. Did you manually added the translation we have npm command in channels/package.json to do it.

File: webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx (hunk 1 above)
Comment (Rajat-Dabade) [reply webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx:31]: We don't have a variable defined by name `--center-channel-color-75`, that's the reason added the fallback.

File: webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx (hunk 2 above)
Comment (Rajat-Dabade) [reply webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx:32]: Yup, removed it.

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by abhijit-singh — COMMENTED** [review]
Thanks @Rajat-Dabade. I believe we should add the labels even when the fields are blank, which will make it look something like this - 

image
 
Currently the labels only appear if there is some content. 

Note that we should still keep the text `Add a channel purpose` and `Add a channel header` clickable in the blank state though. 

@asaadmahmood Can confirm since I think he put together the proposal here.

**Review by hmhealey — APPROVED** [review]
The code looks good pending the changes Abhijit asked about. Feel free to re-request a review if you think the additional changes are big enough to need it

**Comment by asaadmahmood** [comment]
@Rajat-Dabade I agree with @abhijit-singh

**Comment by Rajat-Dabade** [comment]
Updated SC:
Before adding channel purpose and header.
Screenshot 2024-08-01 at 2 24 05 PM

After adding channel purpose and header
Screenshot 2024-08-01 at 2 25 08 PM

cc: @abhijit-singh @asaadmahmood

**Review by abhijit-singh — APPROVED** [review]
Looks great, thanks @Rajat-Dabade!

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**